src/frontend/dash/index.ts

```diff
@@ -23,7 +23,12 @@ function regenCircles(loggedin: LoggedIn) {
         tries++;
         const circles = []
         const now = Date.now()
-        Object.entries(loggedin).forEach(ent => {
+        const loggedInEntries = Object.entries(loggedin)
+        if(loggedInEntries.length > 15)
+            setVisibility(false);
```

Comment:
`setVisibility` make sense when it has the ChiefDelphi context, but when casually looking at the code, it's not super clear what it does. Maybe call it something like `setDelphiVisibility`?

Reply:
That makes sense. I'll add it in